packages/vuetify/src/components/VTooltip/__tests__/VTooltip.spec.cy.tsx

```diff
@@ -11,4 +12,28 @@ const stories = {
 // Tests
 describe('VTooltip', () => {
   generate({ stories, props, component: VTooltip })
+
+  // #issue 15475
+  it('should be activated when activator is on focus and open-on-focus is true', () => {
+    cy.mount(() => (
+      <VBtn>
+        open on focus
+        <VTooltip
+          activator="parent"
+          location="top"
+          open-on-focus
+        >Tooltip</VTooltip>
+      </VBtn>
+    ))
+
+    cy.get('.v-btn').focus()
+
+    // This arbitrary time periods just aims to verify when UI is in stable state.
+    // As there seems to be no way for cypress to wait for an explicit
+    // vuetify/vue internal event finish
+    // eslint-disable-next-line
+    cy.wait(500)
```

Comment:
This "arbitrary time periods" is an anti-pattern in cypress ([link](https://docs.cypress.io/api/commands/wait#Examples)).

I am stuck on finding a good way to allow cypress to wait for an explicit vuetify event (`isTop` watcher logic finish)

Please advice and thanks in advance